src/main/resources/application.yml

```diff
@@ -92,6 +92,7 @@ management:
         http:
           server.requests: true
           client.requests: true
+          DATA_PROCESSING.TimeFromCreationToDeletion: true
```

Comment:
I take it this is the only one that's not got histograms then?

Reply:
Yep